components/graph/git-node.js

```diff
@@ -140,14 +140,10 @@ GitNodeViewModel.prototype.setData = function(logEntry) {
   this.commitComponent.setData(logEntry);
 
   if (logEntry.refs) {
-    var refVMs = logEntry.refs.map(function(ref) {
-      var refViewModel = self.graph.getRef(ref);
-      refViewModel.node(self);
-      return refViewModel;
+    logEntry.refs.map(function(ref) {
```

Comment:
If you do not return anything inside the `map` callback you can use `forEach` which does not create an array.